client/reader/full-post/_style.scss

```diff
@@ -112,7 +112,6 @@
 .reader__full-post-content {
 	@extend %content-font;
 	margin: 0;
-	padding-top: 16px;
```

Comment:
Good call @blowery. I had to add the `!important` rule since the current `.reader__full-post-content` trumps it: https://github.com/Automattic/wp-calypso/pull/8086/commits/643784ecdca03d453fa1f1203da947a532974917